### ISSUE TITLE
Cant paste or type deepgram API key

### ISSUE THREAD

**Issue by techyogi** [8ta4/say#160]
Running latest macos sequioa beta

**Comment by 8ta4** [comment]
Thanks for bringing this issue to my attention, and I'm sorry for the hassle.

I've identified the problem, which turned out to be related to a missing parent directory. I [live-streamed](https://youtu.be/twIcIiJSx_E?t=130) the fix, and everything seems to be [working](https://youtu.be/twIcIiJSx_E?t=139) fine now. I've released [a new version](https://github.com/8ta4/say/releases/tag/v0.9.2) that should take care of this issue.

I guess you could say that the parent directory wasn't the only thing missing. My brain was too!

When you get a chance, please try running [the setup process](https://github.com/8ta4/say/blob/2eb2a67c7ee6e75eb21111fe17ad879ca78256d6/README.md#setup) again, starting with the `brew install 8ta4/say/say` command. If you run into any more problems, just let me know.

**Comment by techyogi** [comment]
That works... Thanks!. Curious and doubt this is the best place to ask, but I was concerned about something i saw on your FAQ about sleeping. If i close my mac's lid it still sleeps, right? If not, a quick way to quit the app that sticks would be helpful... if so, thank you! excited to test this out.

**Comment by 8ta4** [comment]
Yes, closing your Mac's lid will still put it to sleep, just like how reading my lengthy FAQ might put you to sleep!

I'm glad you brought this up, as I've [updated](https://github.com/8ta4/say/commit/4ee5401e88d4a4eb23ace350e3c90e132109d2d1) the FAQ to address your concern.

[Discussions](https://github.com/8ta4/say/discussions/categories/q-a) are the best place to ask questions. Keep the questions coming!
